src/main/java/tr/com/srdc/cda2fhir/transform/ResourceTransformerImpl.java

```diff
@@ -699,16 +706,31 @@ public EntityResult tAssignedAuthor2Practitioner(AssignedAuthor cdaAssignedAutho
 		// representedOrganization -> practitionerRole.managingOrganization
 		if (cdaAssignedAuthor.getRepresentedOrganization() != null
 				&& !cdaAssignedAuthor.getRepresentedOrganization().isSetNullFlavor()) {
-			org.hl7.fhir.dstu3.model.Organization fhirOrganization = tOrganization2Organization(
-					cdaAssignedAuthor.getRepresentedOrganization());
-			fhirPractitionerRole.setOrganization(getReference(fhirOrganization));
-			fhirPractitionerRole.setPractitioner(getReference(fhirPractitioner));
 
+			IEntryResult orgResult = tOrganization2Organization(cdaAssignedAuthor.getRepresentedOrganization(),
+					bundleInfo);
+
+			if (orgResult.hasResult()) {
+				info.setOrgIsNew(true);
+			}
+			result.updateFrom(orgResult);
+			if (orgResult.getFullBundle() != null) {
+				org.hl7.fhir.dstu3.model.Organization fhirOrganization = FHIRUtil
+						.findFirstResource(orgResult.getFullBundle(), org.hl7.fhir.dstu3.model.Organization.class);
```

Comment:
this doesn't seem right to me... why are we just getting the first resource from the bundle?